src/gui/menus/RecentProjectsMenu.cpp

```diff
@@ -0,0 +1,74 @@
+#include "RecentProjectsMenu.h"
+#include "embed.h"
+#include "ConfigManager.h"
+#include <QFileInfo>
+#include "Song.h"
+#include "Engine.h"
+#include "MainWindow.h"
+#include "GuiApplication.h"
+
+RecentProjectsMenu::RecentProjectsMenu(QWidget *parent) :
+	QMenu(tr( "&Recently Opened Projects" ), parent)
+{
+	setIcon(embed::getIconPixmap( "project_open_recent" ));
+
+	connect( this, SIGNAL( aboutToShow() ),
+			 this, SLOT( updateRecentlyOpenedProjectsMenu() ) );
+	connect( this, SIGNAL( triggered( QAction * ) ),
+			 this, SLOT( openRecentlyOpenedProject( QAction * ) ) );
+}
+
+
+
+
+void RecentProjectsMenu::updateRecentlyOpenedProjectsMenu()
```

Comment:
Now that the recent projects menu is in its own class, this method name appears overly descriptive. Maybe try something like `fillMenu()`.

Reply:
sure! :+1: 

My intention was to keep the name the same so it's easier to review but I agree with your comment.